pkg/kudoctl/cmd/uninstall.go

```diff
@@ -0,0 +1,130 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/env"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	uninstallExample = `kubectl kudo uninstall --instance flink`
```

Comment:
could we be more verbose here? Maybe even add example with purge.

Reply:
No longer needed, as the `purge` flag has been removed.